Makefile

```diff
@@ -164,6 +166,14 @@ generate: controller-gen
 	go generate ./...
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+# Check and add (if missing) license header
+LICENSE_FILE=$(shell pwd)/LICENSE
+.PHONY: add-license 
+add-license: addlicense
+	@echo "Checking/Adding license..."
+	@go list -f '{{.Dir}}' ./... | sed -e "s|^$$(pwd)||" | cut -d/ -f2 | sort | uniq | xargs $(ADDLICENSE) -v -f $(LICENSE_FILE) 2>&1
```

Comment:
A couple of suggestions here:
1. Remove the std error redirection here, unless it's needed for something
2. It would be nice to see how `addlicense` is invoked, so we can continue to verify it's working correctly. Could you extract the `go list ... | uniq` pipeline to a variable, and pass that to `addlicense` and remove the echo suppression?

Reply:
Yess thanks a lot! Just added new changes :D

Reply:
Output (if one is missing):

```
echo "Checking/Adding license..."
Checking/Adding license...
/home/thvo/workspace/cryostat-operator/bin/addlicense -v -f /home/thvo/workspace/cryostat-operator/LICENSE api internal
2022/07/04 13:53:50 internal/main.go modified
```